packages/desktop/views/dashboard/buy-sell/components/TransakWindowPlaceholder.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+    import { Platform, openUrlInBrowser } from '@core/app'
+    import { Pane } from '@ui'
+    import { Button, Icon, IconName, Text } from '@bloomwalletio/ui'
+    import { localize } from '@core/i18n'
+    import { DISCORD_URL } from '@contexts/settings/constants'
+
+    let error: boolean = false
+
+    Platform.onEvent('transak-not-loaded', () => (error = true))
+
+    function onButtonClick(): void {
+        openUrlInBrowser(DISCORD_URL)
+    }
+</script>
+
+<Pane
+    classes="flex flex-col justify-center items-center w-full h-full px-6 pb-6 pt-4 gap-4 bg-surface dark:bg-surface-dark shadow-lg"
+>
+    {#if error}
```

Comment:
should we also display something, if we dont have an api key for transak, or could that resulting error only occur in development?